tests/commonTemplates_test.go

```diff
@@ -327,6 +327,17 @@ var _ = Describe("Common templates", func() {
 					updatedTpl.Labels[commonTemplates.TemplateVersionLabel] == "not-latest"
 			}, shortTimeout).Should(BeTrue(), "labels were not removed from older templates")
 		})
+		It("should add deprecated annotation to old templates", func() {
```

Comment:
Test ID CNV-5969

If we add "deprecated" annotation to a new template, will it be reconciled?

Reply:
No. New annotations are not modified by the reconciler.

Reply:
I misunderstood your question.
The new template will be reconciled, but the annotation will not be removed.